demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -302,14 +340,24 @@ def get_new_hooks(
                         ]
                     )
                     hook["args"] = args
+                if should_enable_network(split_by_network, integration_script):
+                    hook["entry"] = hook["entry"].replace("--network none", "")
                 hook[
                     "id"
                 ] = f"{hook['id']}-{integration_script.object_id}"  # for uniqueness
                 hook[
                     "name"
                 ] = f"{hook['name']}-{integration_script.object_id}"  # for uniqueness
+                if change_working_directory:
```

Comment:
cant we set --workdir= argument?

Reply:
we can set `--rootdir`, however in pytest, but still the collection happens in the `cwd`. We have to change the working directory of the container